clearcut_detection_backend/test/scripts/preprocess.py

```diff
@@ -0,0 +1,117 @@
+import os
```

Comment:
can we do separate module, like tiff processing? add there all the needed function for tiff preparation and call them in preprocess stage and also here in preparation. 

But module which is pinned to django will be just connector to the database?